src/modules/hip/handlehip.cpp

```diff
@@ -239,7 +239,7 @@ struct HandleImpl
         }
 
         hipMalloc(&(this->initHandle->mem.mgpu.rgbArr.rgbmem), sizeof(RpptRGB) * this->nBatchSize);
-        hipMalloc(&(this->initHandle->mem.mgpu.scratchBufferHip.floatmem), sizeof(Rpp32f) * 8294400);    // 3840 x 2160
+        hipMalloc(&(this->initHandle->mem.mgpu.scratchBufferHip.floatmem), sizeof(Rpp32f) * 76853888);    // (600000 + 293 + 128) * 128 - Maximum scratch memory required for RNNT training usecase
```

Comment:
Suggest stating what these numbers are - 600000, 293, 128

Reply:
600000 is the maximum size that will be required for MMS buffer based on RNNT dataset
293 is the size required for storing reduction outputs for 600000 sample
128 is the size required for storing cutoff values for batch size 128

Added this details